tools/ts-generator/example/demo.ts

```diff
@@ -35,11 +36,12 @@ const class1: Class1 = {
 };
 
 // Create buffer where the binary data is going to be save
-const buffer = Buffer.alloc(100);
+const io = new IOBuf()
```

Comment:
why is this change in the PR?

Reply:
This change is necessary because I need to fix the `demo` script for running the `Optional` validation in the last commit. 

main problem: the `demo` script isn't updated with the last IObuf implementation.